src/Model/Metadata.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\EuropaSearchClient\Model;
+
+/**
+ * A class that represents a document's metadata.
+ */
+class Metadata
+{
+
+    protected $key;
```

Comment:
sorry for the misunderstanding, I meant not to description, not no property comment, so something like:
```
  /**
   * @var string
   */
```